### DISCUSSION TITLE
An error always occurs when running the model.

### DISCUSSION THREAD

**Discussion by shenjiuye** [NCAR/noahmp#121] [Q&A]
When running the model globally with GLDAS data, I always encounter the problem of 'forrtl: severe (174): SIGSEGV, segmentation fault occurred'. I have tried many methods but still do not know how to solve it. The specific error information is shown in the picture.
![bug](https://github.com/NCAR/noahmp/assets/72537219/2ffd6896-daf2-4f61-b1b2-0bbb3a947e4a)

Has anyone encountered this problem before or knows how to solve it?

**Comment by cenlinhe** [comment]
It looks like there is something wrong with your HRLDAS_setup file. Did you use HRLDAS_forcing folder to generate the HRLDAS_setup file based on GLDAS data? If so, please try re-generating the setup file and forcing files by increasing expand_loop in the namelist.input.GLDAS to a larger number such as 50 or 100. What expand_loop value did you use right now? What is your study domain?  This error may occur over some unresolved islands or coastlines.
